app/ide-desktop/lib/dashboard/src/authentication/src/components/app.tsx

```diff
@@ -110,6 +110,7 @@ function App(props: AppProps) {
 function AppRouter(props: AppProps) {
   const { logger, onAuthenticated } = props;
   const navigate = router.useNavigate();
+  const mainPageUrl = new URL(window.location.href)
```

Comment:
@indiv0 if the `MAIN_PAGE_URL` was just `http://localhost:8080` the this should use `window.location.origin`. href adds the full path. Do we care or is it making a difference